app/controllers/indexes_controller.rb

```diff
@@ -38,7 +38,7 @@ def show
 
     @page_title = @index.name.to_s
     @sectionless = @index.posts.where(index_posts: {index_section_id: nil})
-    @sectionless = @sectionless.order('index_posts.section_order asc')
+    @sectionless = @sectionless.ordered
```

Comment:
This is gonna order by the standard post ordering, which is distinct from the section_order `index_posts` use – can we do a `ordered_by_index` thing instead, which uses this?

Reply:
is that used erroneously anywhere else?

Reply:
Not that I can see! <3

Reply:
followup: https://github.com/Marri/glowfic/pull/680/commits/2835d5d50c7aad697dba88c0f6bdea0d771f61b8#r161358724